pandas/tests/test_resample.py

```diff
@@ -1576,6 +1576,7 @@ def test_resample_categorical_data_with_timedeltaindex(self):
                               'Group': ['A', 'A']},
                              index=pd.to_timedelta([0, 10], unit='s'))
         expected = expected.reindex(['Group_obj', 'Group'], axis=1)
+        expected['Group'] = expected['Group_obj'].astype('category')
```

Comment:
Can you ensure that we have a test with unobserved categories? (i.e. `CategoricalDtype(['A', 'B'])`, and ensure the result dtype has the 'B'.

Reply:
Sure will add that.

Reply:
That's fine, you can use this PR number for the issue.